iStore/Login/LoginView.swift

```diff
@@ -55,15 +47,27 @@ struct LoginView: View {
                     authenticationViewModel.showProgress = true
                     authenticationViewModel.showError = false
                     
-                    authenticationViewModel.authenticateUser(userName: userName, password: password) {
-                        navigationCoordinator.switchView = .contentView
-                        authenticationViewModel.showProgress = false
+                    authenticationViewModel.authenticateUser() { result in
```

Comment:
Why do you call this method in the view?

Reply:
Because It is the action for the login button.

Reply:
Can't see the button here :)

Reply:
you need to look better.